xml_inscriptions/transcribed/IL.Chi.AIC.L.#2024.07.01-mdn.xml

```diff
@@ -0,0 +1,233 @@
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ref type="external" target="https://www.sothebys.com/en/buy/auction/2022/master-sculpture-from-four-millennia/a-roman-marble-strigillated-sarcophagus-1st">Sotheby's auction listing</ref>
+                    </bibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#Hermathena_1907_Abbott"/>
+                        <biblScope unit="item">280-282</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2024-09-22" who="Advaith Rege">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="IL.Chi.AIC.L.#2024.07.01-mdn-entire.jpg"/>
+        </surface>
+        <surface>
+            <desc>Detail of inscription</desc>
+            <graphic url="IL.Chi.AIC.L.#2024.07.01-mdn-det.jpg"/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/>
+                <expan><abbr>D</abbr><ex>is</ex></expan>
+                <expan><abbr>M</abbr><ex>anibus</ex></expan>.
+                <lb n="2"/>
+                <persName>
+                    <name type="praenomen" key="Marcus">
+                        <expan><abbr>M</abbr><ex>arcus</ex></expan>
+                    </name>
+                    <name type="gentilicium" key="Ulpius">Ulpius</name>
+                    <name type="cognomen" key="Cerdo">Cerdo</name>
+                </persName>
+                <lb n="3"/>titulum posuit
+                <persName>
+                    <name type="gentilicium" key="Claudia">Claudiae</name>
+                    <name type="cognomen" key="Tyche">Tycheni</name>
+                </persName>,
+                <lb n="4"/>coniugi
+                <choice><corr>c</corr><sic>k</sic></choice>arissimae,
```

Comment:
I'm not sure we need to correct the k to c here; it's a not infrequent spelling, even if we would say it is a technically incorrect Classical spelling.